app/controllers/application_controller.rb

```diff
@@ -5,9 +5,23 @@ class ApplicationController < ActionController::Base
     if: -> { ENV.key?("BASIC_AUTH_USERNAME") },
   )
 
-  helper_method :signed_in?
+  helper_method :signed_in?, :govuk_verify_enabled?, :current_claim
+
+  private
+
+  def send_unstarted_claiments_to_the_start
```

Comment:
This is still in the ClaimsController. Does it need to be deleted now?

Reply:
good spot!